panels/job_status.py

```diff
@@ -588,6 +595,9 @@ def update_text(self, label, text):
 
     def update_progress (self):
         self.labels['progress_text'].set_text("%s%%" % (str( min(int(self.progress*100),100) )))
+    def update_message (self):
+        msg = self._printer.get_stat("display_status")
+        self.labels['lcdmessage'].set_text("%s" % (msg['message']))
```

Comment:
You may as well use get_stat such as get_stat("display_status","message")

Reply:
Also, instead of `"%s" % (msg['message'])` I would suggest something like `"" if msg == None else msg` so that if there is no message, it doesn't display anything